lib/html_parser.dart

```diff
@@ -792,4 +798,14 @@ class StyledText extends StatelessWidget {
       ),
     );
   }
+
+  double calculateWidth(Display display, RenderContext context) {
+    if ((display == Display.BLOCK || display == Display.LIST_ITEM) && renderContext.parser.shrinkWrap != true) {
```

Comment:
oh super close, you missed a `!= true`. I would just use `!renderContext.parser.shrinkWrap` here

Reply:
I didn't do that in case `shrinkWrap` would be null, but I see now that it defaults to false in the widget constructor so I'll change that - currently working on a different PR so I'm going to try to finish that up and then update this branch.

Reply:
Ah, yes. If it defaulted to null however, I would prefer `!(renderContext.parser.shrinkWrap ?? false)`. In that scenario a `!=` is acceptable, but I think that with null safety coming around we should enforce `??` wherever possible